Win32/Prj/libpcap.vcxproj.filters

```diff
@@ -59,5 +71,29 @@
     <ClInclude Include="..\..\pcap-tc.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\bitset.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\grammar.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\scanner.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="..\..\..\Win32-Extensions\TcApi.h">
```

Comment:
Again, the TurboCap library isn't part of libpcap/WinPcap/Npcap, it's a separate library provided by Riverbed, so it should be looking for that header externally.